### ISSUE TITLE
Use run_lm-finetuning on tpu

### ISSUE THREAD

**Issue by abdallah197** [huggingface/transformers#2089]
## ❓ Questions & Help

Is it possible to use the script run_lm-finetuning on TPUs, if not, what do you recommend to fine-tune BERT language model on TPUs using the transformers library

**Comment by LysandreJik** [comment]
Hello, the script would need to be adapted to run on TPU to take full advantage of the chips. We're actively working with the Cloud TPU team on scripts for fine-tuning on TPUs, which should be available in the coming weeks.
